### PR TITLE
replace findinotify with KDE's version

### PR DESCRIPTION
**Pull request by neheb**
Seems to be better maintained.

Modified to support SunOS as done before.

Signed-off-by: Rosen Penev <rosenp@gmail.com>

### CODE REVIEW

**Comment by whyman** [comment]
Would be nice to have a target if we are going to change it

**Comment by neheb** [comment]
KDE's version does not have a target.

OTOH, it's available on most distros as extra-cmake-modules or some variation.

**Comment by neheb** [comment]
Anyway, https://invent.kde.org/frameworks/extra-cmake-modules/-/merge_requests/209

**Comment by neheb** [comment]
I added a target thing. No idea if I did it right. Copy paste from the iconv module.

File: cmake/FindInotify.cmake [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                Inotify_INCLUDE_DIRS
+        )
+        mark_as_advanced(Inotify_LIBRARIES Inotify_INCLUDE_DIRS)
+        include(FeatureSummary)
+        set_package_properties(Inotify PROPERTIES
+            URL "https://github.com/libinotify-kqueue/"
+            DESCRIPTION "inotify API on the *BSD family of operating systems."
+        )
+    endif()
+  if(NOT TARGET Inotify::Inotify)
```
Comment (whyman) [inline cmake/FindInotify.cmake:57]: Can you fix the whitespace?

Looks good

Comment (neheb) [reply cmake/FindInotify.cmake:57]: What whitespace?

**Pull request merged**